.github/CODEOWNERS

```diff
@@ -0,0 +1 @@
+docs/style-guide.rst @fulldecent
```

Comment:
And another point from that article:

> While GitHub suggests *write* permissions, in practice I’ve found that *maintain* permissions are what you really need. The whole idea of using a `CODEOWNERS` file is to leverage GitHub’s *protected* branches feature where a Code Owner must review a PR before it can be merged to the protected branch. Maintain permissions seem to be required to hit that merge button on a protected branch, this comes in handy when a Code Owner complains “Why can’t I merge this PR that I just approved?”.

It apprears that being in `CODEOWNERS` will not do much beyond automatically adding @fulldecent as a reviewer. It won't require his review though or won't let him merge code style PRs on his own. Is that enough of should we figure out an alternative that will give him more control over these docs?

Reply:
Looks like this `CODEOWNERS` thing works differently than we imagined it. See for example [Is there any way to add people that dont have write permissions fo the repository as CODEOWNERS?](https://github.community/t/is-there-any-way-to-add-people-that-dont-have-write-permissions-fo-the-repository-as-codeowners/2015), which has an official github response. Looks like the file is more of a mechanism to put even more restrictions on some files rather than alter or lift some restrictions for specific people.

Reply:
So we leave it be?

Reply:
Thank you. Good as-is now. I'll get pinged which is important.

Reply:
> So we leave it be?

I'd say that this would be a good reason for having a separate repo after all.

But since @fulldecent says he's fine with it, I guess we can continue as is. I'm going to approve this and we can revisit when that lack of greater permissions starts getting in the way.